### PR TITLE
Allow providing custom views as accessory

### PR DESCRIPTION
**Pull request by ilyapuchka**
This will allow to provide a custom view, i.e. button, as accessory view. Here it's a button with "call" title and icon on the right side.

screen shot 2018-11-15 at 18 21 57

### CODE REVIEW

File: BentoKit/BentoKit/Views/Accessory.swift [hunk 1]
```diff
@@ -42,6 +42,10 @@ public final class AccessoryView: InteractiveView {
     }
 
     public override var intrinsicContentSize: CGSize {
+        if case let .custom(view) = accessory {
+            view.layoutIfNeeded()
```
Comment (sergdort) [inline BentoKit/BentoKit/Views/Accessory.swift]: I think if `view` would pinned to the edges not to `centerInSuperview` them you would not need that

Comment (ilyapuchka) [reply BentoKit/BentoKit/Views/Accessory.swift]: what about width?
Actually I think there needs to be an adjustment for the height calculation as it currently has some constant `24`. Though it seems to be working in playground correctly.

Comment (andersio) [reply BentoKit/BentoKit/Views/Accessory.swift]: Why isn't the intrinsic content size of the custom view be used? If one needs an accessory that cannot have a definitive intrinsic content size, it sounds something is off.

**Comment by ilyapuchka** [comment]
@andersio When I tried to return just this view intrinsic content size does not work, i.e. if one tries to use a UIButton. Though this was my original idea. I'll add a screenshot.

**Comment by ilyapuchka** [comment]
In fact this is what happening in test with `return view.intrinsicContentSize`

screen shot 2018-11-16 at 18 48 37

And here correct layout with `view.layoutIfNeeded`

screen shot 2018-11-16 at 18 48 15

File: BentoKit/BentoKit/Views/Accessory.swift (hunk 1 above)
Comment (andersio) [reply BentoKit/BentoKit/Views/Accessory.swift]: Fair enough.

**Pull request merged**